package.json

```diff
@@ -23,10 +23,10 @@
     "highlight"
   ],
   "license": "Unlicense",
-  "repository": "https://github.com/reed-tf2/language-tf2-cfg",
+  "repository": "https://github.com/XcapeEst/language-tf2-cfg",
   "bugs": {
-    "url": "https://github.com/reed-tf2/language-tf2-cfg/issues",
-    "email": "reed.tf2@gmail.com"
+    "url": "https://github.com/XcapeEst/language-tf2-cfg/issues",
+    "email": "mark.veiermann@gmail.com"
```

Comment:
Once we get this PR figured out, I'm planning to re-write my commit authorship to remove my real name. You might prefer to use an alias email too?